### PR TITLE
Accessibility: missing aria-label attribute in "socialicon" elements

### PR DESCRIPTION
**Pull request by nadavkav**
A11Y

### CODE REVIEW

**Comment by gjb2048** [comment]
Not needed as already has 'title' attribute.

**Comment by nadavkav** [comment]
It seems so, reading the spec, but I still got errors while checking it with WAVE and AXE.
Adding the aria-label, which precedes "title", solved it.
Your call, by I have to keep it as we are being audited for accessibility and it shows on the report.
(Didn't check it yet with a proper screen reader)

**Comment by gjb2048** [comment]
Ok then, need screen shot evidence :)

**Comment by gjb2048** [comment]
Hi Nadav,

Please provide screen shot evidence of report to convince me.

G

**Comment by nadavkav** [comment]
I will

**Comment by gjb2048** [comment]
Challenge accepted? :)

**Comment by nadavkav** [comment]
This is WAVE report, after taking the "fix" off:
image

**Comment by nadavkav** [comment]
(btw, I was asked to move the social icons to the footer, from the header. but this was also happening on the header, of course)

File: classes/output/core_renderer.php [hunk 1]
```diff
@@ -1736,6 +1736,7 @@ public function render_social_network($socialnetwork) {
                 'class' => 'socialicon ' . $socialnetwork,
                 'onclick' => "window.open('".\theme_essential\toolbox::get_setting($socialnetwork)."')",
                 'title' => get_string($socialnetwork, 'theme_essential'),
+                'aria-label' => $socialnetwork,
```
Comment (gjb2048) [inline classes/output/core_renderer.php]: Rejected, as using the 'key' and not the result of the get_string on that key.  Please refactor and resubmit as a single commit.

Comment (nadavkav) [inline classes/output/core_renderer.php]: Right. I updated the patch.

**Comment by gjb2048** [comment]
Thanks Nadav, snag is the solution is inefficient as its calling get_string twice for the same thing.  Better to store the result and reuse.

**Comment by nadavkav** [comment]
Right again :smile: (updating...)

**Comment by nadavkav** [comment]
Updated

**Pull request merged**